plugins/org.eclipse.n4js/src/org/eclipse/n4js/scoping/N4JSScopeProvider.xtend

```diff
@@ -158,6 +158,9 @@ class N4JSScopeProvider extends AbstractScopeProvider implements IDelegatingScop
 	/** True: Proxies of IdentifierRefs are only resolved within the resource. Otherwise, the proxy is returned. */
 	private boolean suppressCrossFileResolutionOfIdentifierRef = false;
 	
+	/** True: Request will return all members according to receiver type and ignore whether read/write access restriction. */
```

Comment:
Wording is odd ("and ignore whether ... restriction").

Also: is this only about read access vs. write access or also about  typing strategy, i.e. `~r~` vs. `~w~` etc. ? In the latter case, it would be good to mention this in the comment as well.

EDIT: after looking at code below, probably typing strategy does not have to be mentioned here.